src/fhirtypes/CodeSystem.ts

```diff
@@ -52,7 +53,7 @@ export class CodeSystem {
    * @returns {string} the filename
    */
   getFileName(): string {
-    return `CodeSystem-${this.id}.json`;
+    return sanitize(`CodeSystem-${this.id.replace(/\/\\/g, '-')}.json`, { replacement: '-' });
```

Comment:
Cool! Handy dandy suggestion to fix it!

```suggestion
    return sanitize(`CodeSystem-${this.id}.json`, { replacement: '-' });
```